airbyte-ci/connectors/pipelines/pipelines/cli/airbyte_ci.py

```diff
@@ -174,64 +232,22 @@ def get_modified_files(
 )
 @click.option("--ci-job-key", envvar="CI_JOB_KEY", type=str)
 @click.option("--show-dagger-logs/--hide-dagger-logs", default=False, type=bool)
-@click.pass_context
-@track_command
+@click_track_command
+@click_merge_args_into_context_obj
```

Comment:
@alafanechere @erohmensing This is what merges all click arguments/options into the context object by default.

basically another name would be `@pass_all_click_args_to_child`

Reply:
I saw this in use before I got down to my comment. This is sexy 🔥